frontend/src/metabase/admin/settings/components/SettingsUpdatesForm/SettingsUpdatesForm.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import MetabaseSettings from "metabase/lib/settings";
+import SettingsSetting from "../SettingsSetting";
+
+import VersionUpdateNotice from "./VersionUpdateNotice/VersionUpdateNotice";
+
+export default class SettingsUpdatesForm extends Component {
```

Comment:
I wouldn't be opposed to making this a function component since there's only a `render` method